modules/ob-disabling-shipwrightbuilds-build-controller-webhook-pod.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
```

Comment:
The section about disabling Shipwright is not related to the Shared Resource CSI Driver. It should get moved to the "Install" page.